template/web/gatsby-node.js

```diff
@@ -46,6 +44,70 @@ async function createBlogPostPages (graphql, actions) {
     })
 }
 
+async function createCategoryPages (graphql, actions) {
+  const {createPage} = actions
+
+  const result = await graphql(`
+    {
+      allSanityCategory {
+        edges {
+          node {
+            id
+            slug {
+              current
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  if (result.errors) throw result.errors
+
+  const categoryNodes = (result.data.allSanityCategory || {}).edges || []
```

Comment:
Maybe we should use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) instead?